.github/actions/check-codescanning-config/action.yml

```
@@ -29,7 +29,15 @@ inputs:
   tools:
     required: true
     description: |
-      The url of codeql to use.
+      The version of the CodeQL tools to use. This can be any of the following:
```

This is an action used internally for testing only. Maybe better to change this to:

```suggestion
      The version of CodeQL passed to the `tools` input of the init action.
```